example/package.json

```diff
@@ -8,10 +8,21 @@
       "types": "./dist/types.d.mts",
       "default": "./dist/module.mjs"
     },
-    "./utils": "./dist/utils.mjs"
+    "./utils": {
+      "types": "./dist/utils.d.mts",
```

Comment:
is this needed? I think we can omit `.mts` for subpaths that match the types and typescript will infer them

Reply:
done